Streetcode/Streetcode/Program.cs

```diff
@@ -1,14 +1,30 @@
+using Microsoft.AspNetCore.Hosting;
 using Microsoft.EntityFrameworkCore;
 using Repositories;
 using Services.Interfaces;
 using Services.Services;
+using System;
+
+// for Nuke
+//environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+//if (environment != "Local")
+//{
+//    Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Local", EnvironmentVariableTarget.Machine);
+//}
```

Comment:
You can remove these comments, these should be in Nuke SetupEnvironmentVariables target